scriptmodules/supplementary/pegasus-fe.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="pegasus-fe"
+rp_module_desc="Pegasus: A cross platform, customizable graphical frontend (latest alpha release)"
+rp_module_licence="GPL3+ https://raw.githubusercontent.com/mmatyas/pegasus-frontend/master/LICENSE.md"
+rp_module_section="exp"
+rp_module_flags="!mali frontend"
+
+function depends_pegasus-fe() {
+    local depends=(
+        gstreamer1.0-alsa
+        gstreamer1.0-plugins-good
+        jq
+        libinput10
+        libsdl2-2.0-0
```

Comment:
Although this is a binary only install we use the dev packages for dependencies - this is due to the names of binary packages changing - so please just include libsdl2-dev etc.

This doesn't currently work on my x86_64 box as the binary it installs is linked against libpng12 which isn't available on Ubuntu 17.04. Perhaps if distributing a binary only, you can statically link the png library ?